release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -80,21 +80,24 @@ For more information, see xref:../backup_and_restore/hibernating-cluster.adoc#hi
 [id="ocp-release-notes-extensions_{context}"]
 === Extensions ({olmv1})
 
-Operator Lifecycle Manager (OLM) has been included with {product-title} 4 since its initial release and has helped grow and enable a substantial ecosystem of solutions and advanced workloads running as Operators. {product-title} {product-version} introduces {olmv1}, the next-generation Operator Lifecycle Manager, as a General Availability feature, designed to improve how you manage Operators on {product-title}.
+[id="ocp-release-notes-extensions-olmv1-ga_{context}"]
+==== {olmv1-first} (General Availability)
+
+Operator Lifecycle Manager (OLM) has been included with {product-title} 4 since its initial release and has helped enable and grow a substantial ecosystem of solutions and advanced workloads running as Operators. {product-title} {product-version} introduces {olmv1}, the next-generation Operator Lifecycle Manager, as a General Availability (GA) feature, designed to improve how you manage Operators on {product-title}.
 
 Previously available as a Technology Preview feature only, this updated framework evolves many of the concepts that have been part of previous versions of {olmv0} by simplifying Operator management, enhancing secrutiy, and boosting reliability.
 
 [NOTE]
 ====
-The documentation for {olmv1} exists as a stand-alone guide called xref:../extensions/index.adoc#olmv1-about[Extensions]. Previously, {olmv1} documentation was a subsection of the xref:../operators/index.adoc#operators-overview[Operators] guide, which otherwise documents the {olmv0} feature set.
+Starting in {product-title} {product-version}, {olmv1} is now enabled by default, alongside the {olmv0}. Cluster administrators can toggle installation of {olmv1} by using cluster capabilities at installation time.
```

Comment:
Can we use `capability` instead of  `toggle` here. OLMv1 is a capability which users can disable before installation of OCP https://docs.openshift.com/container-platform/4.17/installing/overview/cluster-capabilities.html